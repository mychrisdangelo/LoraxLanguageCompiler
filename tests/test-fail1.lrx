/* 
 * Author: Zhaarn Maheswaran
 * Checks that semantic analysis catches type mismatch 
 */

int main () {
	print(1 + 1.0);
}
