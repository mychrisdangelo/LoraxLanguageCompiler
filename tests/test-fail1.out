Fatal error: exception Failure("operator + not compatible with expressions of type int and float")
